tasks/modprobe.yml

```diff
@@ -1,3 +1,22 @@
 ---
 
-# modprobe
+- name: check if modprobe is already installed | os-10
+  command: 'modprobe -V'
+  register: modprobe_installed
+  ignore_errors: True
+  changed_when: False
```

Comment:
I would drop this check task. The next task can just ensure that `kmod` is present.